lib/puppet/provider/gluster.rb

```diff
@@ -1,3 +1,6 @@
+require 'rexml/document'
```

Comment:
It was working by accident before, probably because something in puppet that used REXML either doesn't use it anymore or doesn't get pulled in during an agent run. (The tests still pass without the require even though the puppet run fails, which is a little annoying.)